packages/angularjs-auth-passport/lib/webAuthService.js

```diff
@@ -104,4 +115,16 @@ WebAuthService.prototype.logout = function() {
   });
 }
 
+
+/**
+ * Private - show dialog when cloudUrl is missing
+ */
+WebAuthService.prototype.showAlertDialog = function() {
```

Comment:
Maybe give this method a more descriptive name or generic callout, such as `showMissingCloudUrlDialog()` or `showAlertDialog(new Error('Application is missing cloud server url. Please see documentation for proper configuration'))`